k8s/service_controller/server/k8s/storage.go

```diff
@@ -44,16 +45,36 @@ func (kss *K8sServiceStorage) ListBrokers() ([]*model.ServiceBroker, error) {
 		return nil, fmt.Errorf("couldn't get the service brokers. %v, [%v]", e, s)
 	}
 
+	fmt.Println("returned json: ", s)
+
 	var lsb listSB
 	e = json.Unmarshal(b, &lsb)
 	if nil != e { // wrong json format error
+		fmt.Println("json not unmarshalled:", e, s)
 		return nil, e
 	}
-	return lsb.Items, nil
+	fmt.Println("returning %v", lsb.Items)
+	ret := make([]*model.ServiceBroker, 0)
+	for _, v := range lsb.Items {
+		ret = append(ret, v.ServiceBroker)
+	}
+	return ret, nil
 }
 
-func (kss *K8sServiceStorage) GetBroker(id string) (*model.ServiceBroker, error) {
-	return nil, fmt.Errorf("Not implemented yet")
+func (kss *K8sServiceStorage) GetBroker(name string) (*model.ServiceBroker, error) {
```

Comment:
Was this a bad merge? At this level the broker key is going to be the GUID rather than broker name.